distributed/cli/dask_spec.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import click
+import json
+import yaml
+
+from distributed.deploy.spec import run_spec
+
+
+@click.command(context_settings=dict(ignore_unknown_options=True))
+@click.argument("args", nargs=-1)
+@click.option("--spec", type=str, default="", help="")
+@click.option("--spec-file", type=str, default=None, help="")
+@click.version_option()
+def main(args, spec: str, spec_file: str):
```

Comment:
Do you allow both `spec` and `spec-file`? I'd make them a `xor` relationship, should be only one or the other. I think mixing would be confusing and don't really see a use case for it.

Reply:
Good ideas both.  I'll handle this tomorrow.  Thanks!